README.md

```diff
@@ -7,7 +7,8 @@
 
 > Self identifying base encodings
 
-Multibase is a protocol for distinguishing base encodings of bytes in text.
+Multibase is a protocol for disambiguating the encoding of base-encoded (e.g.,
+base32, base64, base58, etc.) binary appearing in text.
```

Comment:
I like this one better. What about "…binary represented as strings"? I find "appearing in text" awkward to read. For me as programmer "strings" are a concept I'm used to. I can already smell that somehow character encoding will also be a topic in this spec.

Though if "strings" are not technically correct, I'm happy to keep the current text.

Reply:
We can also change it to "in text" instead of "appearing in text" but, IMO, "strings" is still ambiguous.

Reply:
I like "appearing in text" better, so let's keep like that.